util.py

```diff
@@ -1,5 +1,44 @@
 import os
 import errno
+import logging
+from collections import namedtuple
+from os.path import basename, dirname, splitext, join
+from glob import glob
+
+
+SimplePath = namedtuple('Path', ['dir', 'name', 'ext'])
+
+
+class Path(SimplePath):
+    def read(self):
+        with open(str(self), 'r') as f:
+            return f.read()
+
+    def __str__(self):
+        return(join(*self[:2]) + self.ext)
+
+    def rename(self, name):
+        new_path = Path(self.dir, name, self.ext)
+        os.rename(str(self), str(new_path))
+        return new_path
+
+
+def get_function(section, config):
+    module_name = config.get(section, 'module')
+    fun_name = config.get(section, 'function')
+    module = __import__(module_name, globals(), locals(), [], 0)
+    return getattr(module, fun_name)
+
+
+def update_config(config, update_str):
+    for k, v in (e.split('=') for e in update_str.split(',') if e):
+        config.set('score', k, v)
```

Comment:
This looks weird. Should it just be `config.set(k, v)` ?

Reply:
Oh ok, so I think I get it now, however `'score'` and `'solve'` should be passed into `update_config`, right? Because now we update the `'score'`-config with the `--solve key1=value1` parameter?

Reply:
Good catch, will fix this